docs/sources/whatsnew/whats-new-in-v8-0.md

```diff
@@ -73,7 +73,8 @@ For more information, refer to [State timeline visualization]({{< relref "../pan
 
 ### Status history visualization (beta)
 
-A sister panel to the state timeline is the new Status history panel visualization. It can display periodic state in a grid. Works with both numerical, string or boolean state.
+A sister panel to the state timeline is the new Status history panel visualization. It can display periodic state in a grid. It supports both numerical, string or boolean state. Colors can be assigned using value mappings, thresholds or gradient color
```

Comment:
```suggestion
A sister panel to the state timeline is the new Status history panel visualization. It can display periodic states in a grid view. It supports both numerical, string, or boolean states. You can assign colors using value mappings, thresholds, or gradient color. For more information, refer to [Status histor]({{< relref "../panels/visualizations/status-history.md" >}}).
```